src/main/java/atmosfears/AtmosFearsBE/controller/DataIngestController.java

```diff
@@ -0,0 +1,39 @@
+package atmosfears.AtmosFearsBE.controller;
+
+import atmosfears.AtmosFearsBE.model.AirParticulatesFile;
+import atmosfears.AtmosFearsBE.service.AirParticulatesService;
+import atmosfears.AtmosFearsBE.service.CalculationService;
+import atmosfears.AtmosFearsBE.service.DataLoadService;
+import java.util.Set;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@RequestMapping("/api/ingest")
+public class DataIngestController {
+
+  @Autowired
+  DataLoadService dataLoadService;
+
+  @Autowired
+  CalculationService calculationService;
+
+  @Autowired
+  AirParticulatesService airParticulatesService;
+
+  @PostMapping("/load")
```

Comment:
```suggestion
  @PostMapping("/")
```

Don't use verbs for endpoint names. Following REST guidelines, we shouldn't use words indicating an action. 

If there are no more endpoints with `@PostMapping` you can use the `/` path. It might be changed later if more POST mapping endpoints are added.